pom.xml

```diff
@@ -31,52 +32,52 @@
 		<selenium.version>2.45.0</selenium.version>
 	</properties>
 
-    <repositories>
-        <repository>
-            <id>devhub</id>
-            <url>https://devhub.ewi.tudelft.nl/nexus/content/repositories/releases/</url>
-            <releases>
-                <enabled>true</enabled>
-                <updatePolicy>always</updatePolicy>
-            </releases>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-        </repository>
-    </repositories>
+	<repositories>
```

Comment:
This was probably a side effect of indenting the checkstyle-plugin.

Spaces vs tabs should be a seperate issue I think as it will effect the whole code. When I initially enabled this Checkstyle rule, 6000 CheckStyle warnings were generated.